examples/src/java/org/partiql/examples/EvaluationJavaExample.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at:
+ *
+ *      http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+
+package org.partiql.examples;
+
+import com.amazon.ion.IonSystem;
+import com.amazon.ion.system.IonSystemBuilder;
+import kotlin.Unit;
+import org.jetbrains.annotations.NotNull;
+import org.partiql.examples.util.Example;
+import org.partiql.lang.CompilerPipeline;
+import org.partiql.lang.eval.Bindings;
+import org.partiql.lang.eval.EvaluationSession;
+import org.partiql.lang.eval.ExprValue;
+import org.partiql.lang.eval.Expression;
+import org.partiql.lang.syntax.SqlParser;
+
+import java.io.PrintStream;
+
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonMap;
+
+public class EvaluationJavaExample extends Example {
+
+    public EvaluationJavaExample(@NotNull PrintStream out) {
+        super(out);
+    }
+
+    @Override
+    public void run() {
+
+        // A standard instance of [IonSystem], which is required by [CompilerPipeline].
+        final IonSystem ion = IonSystemBuilder.standard().build();
+
+        // An instance of [CompilerPipeline].
+        final CompilerPipeline pipeline = CompilerPipeline.standard(ion);
+
+        // Compiles a simple expression containing a reference to a global variable.
+        final String query = "'Hello, ' || user_name";
+        print("PartiQL query:", query);
+        final Expression e = pipeline.compile(query);
+
+        // This is the value of the global variable.
+        final String userName = "Homer Simpson";
+        final ExprValue usernameValue = pipeline.getValueFactory().newString(userName);
+
+        // [Bindings.ofMap] can be used to construct a [Bindings] instance of
+        // bindings with previously materialized values.
+        final Bindings<ExprValue> globalVariables = Bindings.ofMap(singletonMap("user_name", usernameValue));
+        print("global variables:", "user_name => userName");
```

Comment:
Should we print the `globalVariables`?

Reply:
Yes. This is what was in the Kotlin example. I will fix both.